src/EditorFeatures/CSharpTest/AvoidUnusedMembers/AvoidUnusedMembersTests.cs

```diff
@@ -0,0 +1,1394 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.AvoidUnusedMembers;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.AvoidUnusedMembers;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.AvoidUnusedMembers
```

Comment:
the name of this feature does not match our names for other similar features.  Should probably be RemoveUnusedMembers.  Like RemoveUnusedImports RemoveUnusedVariable, etc.